packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.base.tsx

```diff
@@ -246,4 +234,25 @@ export class HoverCardBase extends BaseComponent<IHoverCardProps, IHoverCardStat
       return prevState;
     });
   };
+
+  private _setEventListeners = (): void => {
+    const { trapFocus, instantOpenOnClick } = this.props;
+    const target = this._getTargetElement();
+    const nativeEventDismiss = this._cardDismiss.bind(this, true);
```

Comment:
I understand rebinding the `target` event listeners on `target` change, but do we need to rebind this function?